src/plugins/intel_cpu/src/emitters/jit_load_store_emitters.cpp

```diff
@@ -1014,19 +1018,39 @@ void jit_store_emitter::store_dword_to_word_extension(const Vmm &vmm, const Xbya
     };
 
     if (is_bf16) {
-        // to avoid src vmm pollution
-        if (src_prc_ == Precision::FP32) {
-            ymm = Ymm(aux_vec_idxs[0]);
-        }
-        if (mayiuse(cpu::x64::avx512_core_bf16)) {
-            h->vcvtneps2bf16(ymm, zmm);
-        } else {
-            emu_vcvtneps2bf16_->emit_code({static_cast<size_t>(vmm.getIdx())}, {static_cast<size_t>(ymm.getIdx())});
-        }
-        if (store_num == 16) {
-            h->vmovdqu16(ptr[reg + offset], ymm);
+        if (mayiuse(cpu::x64::avx512_core)) {
+            // to avoid src vmm pollution
+            if (src_prc_ == Precision::FP32) {
+                ymm = Ymm(aux_vec_idxs[0]);
+            }
+            if (mayiuse(cpu::x64::avx512_core_bf16)) {
+                h->vcvtneps2bf16(ymm, zmm);
```

Comment:
Looks like we now can move `vcvtneps2bf16` inside the emitter itself and rename it to `jit_uni_vcvtneps2bf16`, wo it will correctly work for each ISA.